java-frontend/src/test/java/org/sonar/java/model/LiteralUtilsTest.java

```diff
@@ -184,11 +184,13 @@ public void is_0xff() {
     assertThat(LiteralUtils.is0xff(tree)).isFalse();
   }
 
+  @Test
```

Comment:
Ouch! Can we think of a new rule to catch that?

Reply:
Public non-static method in test class, without invocations from within the file, and having assertions but no test annotations?

Reply:
@m-g-sonar Let's evaluate that pattern on our code base :)